config/locales/it.yml

```diff
@@ -156,9 +156,9 @@ it:
         system_status: Stato del sistema
       users:
         destroy:
-          confirmation: Sei sicuro di voler cancellare l'utente "%{user}"? Questa azione è irreversibile.
-          failure: L'utente "%{user}" non può essere cancellato.
-          success: L'utente "%{user}" è stato cancellato con successo.
+          confirmation: Sei sicuro di voler cancellare l'utente '%{user}'? Questa azione è irreversibile.
+          failure: L'utente '%{user}' non può essere cancellato.
+          success: L'utente '%{user}' è stato cancellato con successo.
```

Comment:
Because of the use of a single quote in this string, I think you will need to use double quotes here